wordpress-mobile/WordPress-Android
# Dashboard domain card UI

Fixes #18178 

<img width=320 src="https://user-images.githubusercontent.com/990349/230291552-4864c51c-e672-427f-8b38-c136efe48405.png" />

To test:

1. Launch Jetpack/WordPress app
2. Go to **App Settings** (Tap on Avatar at the top right hand corner on My Site -> Find App Settings)
3. Tap **Debug Settings**
4. Find **dashboard_card_domain** under Remote features as shown in the image above
5. Verify **domain card** appears as shown above on dashboard
6. Tap on overflow (three dots) menu
7. **Verify** it pops up menu with **Hide this** option
8. Tap on Hide this
9. **Verify**, the card gets hidden 

>** Note **
> The 🌐 domain icon needs to be updated, waiting for export.  Will update before merging or in the next PR

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Unit tests and manual tests

3. What automated tests I added (or what prevented me from doing so)
Updated and added unit tests

PR submission checklist:
